target/linux/ramips/dts/mt7621_ubnt_unifi-flexhd.dts

```diff
@@ -61,6 +61,20 @@
 				label = "factory";
 				reg = <0x70000 0x10000>;
 				read-only;
+
+				nvmem-layout {
+					compatible = "fixed-layout";
+					#address-cells = <1>;
+					#size-cells = <1>;
+
+					eeprom_factory_0: eeprom@0 {
+						reg = <0x0 0x400>;
+					};
+
+					eeprom_factory_8000: eeprom@8000 {
+						reg = <0x8000 0x4da8>;
```

Comment:
Is this correct?

Reply:
Hi! Thanks for your review.

I think it's right. This device uses MT7603 + MT7615.
https://github.com/openwrt/openwrt/blob/a8cf9f860f1a98f7cfe8547710df77f73ec5a3d4/target/linux/ramips/image/mt7621.mk#L2538

It would be easier to review these commits one by one. I have added the NIC models of each device/vendor in the information.

Reply:
Very good work. I'm merging this hoping everything goes well. Lets see.